pkg/csi/service/wcpguest/controller_helper.go

```diff
@@ -160,23 +161,32 @@ func checkForSupervisorPVCCondition(ctx context.Context, client clientset.Interf
 		if !ok {
 			continue
 		}
-		if checkPVCCondition(ctx, pvc, reqCondition) {
+		if checkPVCCondition(ctx, pvc, reqCondition, reqSize) {
 			return nil
 		}
 	}
-	return fmt.Errorf("supervisor persistentVolumeClaim %s in namespace %s not in %q condition within %d seconds",
-		pvcName, ns, reqCondition, timeoutSeconds)
+	return fmt.Errorf("supervisor PersistentVolumeClaim %s in namespace %s not in %s condition and %s size "+
+		"within %d seconds", pvcName, ns, reqCondition, reqSize.String(), timeoutSeconds)
 }
 
 func checkPVCCondition(ctx context.Context, pvc *v1.PersistentVolumeClaim,
```

Comment:
Can you add method doc for `checkPVCCondition`?

Reply:
done